frontend/src/components/pages/UserProfilePage.tsx

```diff
@@ -1,18 +1,162 @@
+        }`,
+      );
+    },
+    onError: () => {
+      history.push("/404");
+    },
+  });
+
+  const [deleteUser] = useMutation<{
+    response: SoftDeleteUserResponse;
+  }>(SOFT_DELETE_USER);
+
+  const closeModal = () => {
+    setConfirmDeleteUser(false);
+  };
+
+  const openModal = () => {
+    setConfirmDeleteUser(true);
+  };
+
+  const callSoftDeleteUserMutation = async () => {
+    await deleteUser({
+      variables: {
+        id: userId,
+      },
+    });
+    closeModal();
+    history.push("/");
+  };
+
+  const filterStyle = useStyleConfig("Filter");
   return (
-    <Box textAlign="center">
-      <h1>User Profile for {userId} :)</h1>
-    </Box>
+    <Flex direction="column" height="100vh">
+      <Header />
+      <Flex direction="row" flex={1}>
+        <Flex sx={filterStyle} maxWidth="400px">
+          <Heading size="lg">{fullName}</Heading>
+          <Heading size="sm" marginTop="36px">Role(s)</Heading>
+          <Text>{role}</Text>
+          <Heading size="sm" marginTop="36px">Email</Heading>
+          <Text>{email}</Text>
+          <Heading size="sm" marginTop="36px">Date joined</Heading>
+          <Text>TODO</Text>
+          <Heading size="sm" marginTop="36px">Last active on</Heading>
+          <Text>TODO</Text>
+        </Flex>
+        <Flex direction="column" margin="40px">
+          <Heading size="lg" marginTop="40px">
+            Approved Languages and Levels
+          </Heading>
+          <Text>TODO: table here</Text>
+          <Heading size="lg" marginTop="56px">
+            Assigned Story Translations
+          </Heading>
+          <Text>TODO: table here</Text>
+          <Heading size="sm" marginTop="56px">
+            Delete user
+          </Heading>
+          <Text>
+            Permanently delete this user and all data associated with them.
+          </Text>
+          <Button
+            colorScheme="red"
+            margin="10px 0px"
+            width="250px"
+            onClick={() => openModal()}
+          >
+            Delete User
+          </Button>
+        </Flex>
+      </Flex>
+      <Flex
+        alignItems="center"
+        boxShadow="0 0 12px -9px rgba(0, 0, 0, 0.7)"
+        direction="row"
+        height="90px"
+        justify="flex-end"
+        padding="20px 30px"
+      >
+        <Box>
+          <Button colorScheme="blue" variant="outline">
+            Cancel
+          </Button>
+          {/* TODO: make this functional.*/}
+          <Button colorScheme="blue" margin="0 48px 0 24px">
+            Save Changes
+          </Button>
+        </Box>
+      </Flex>
+      {confirmDeleteUser && (
+        <ConfirmationModal
+          confirmation={confirmDeleteUser}
+          onClose={closeModal}
+          onConfirmationClick={callSoftDeleteUserMutation}
+          confirmationMessage={MANAGE_USERS_TABLE_DELETE_USER_CONFIRMATION}
+          buttonMessage={MANAGE_USERS_TABLE_DELETE_USER_BUTTON}
+        />
+      )}
+    </Flex>
```

Comment:
nit: The way the panels (i.e. sidebar and bottom bar with the cancel and save buttons) overlap is different from how they overlap on Figma. The difference is pretty subtle, so not sure if you need to change it.

Reply:
feel free to do whatever makes it consistent with manage story translation page (cant tell from just looking at it)